llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -52,108 +61,118 @@ class RISCVExperimentalExtension<string name, int major, int minor, string desc,
 
 def FeatureStdExtI
     : RISCVExtension<"i", 2, 1,
-                     "'I' (Base Integer Instruction Set)">;
+                     "'I' (Base Integer Instruction Set)">,
+                     RISCVExtensionBitmask<0, 0>;
```

Comment:
I think we should align `RISCVExtensionBitmask` with `RISCVExtension`.

Reply:
Aligned.